test/mongoose_push_web/controllers/api_v3_notification_controller_test.exs

```diff
@@ -27,7 +27,7 @@ defmodule MongoosePushWeb.APIv3NotificationControllerTest do
         Jason.encode!(Map.drop(ControllersHelper.alert_request(), ["service"]))
       )
 
-    assert json_response(conn, 422) == ControllersHelper.no_schemas_provided_response()
+    assert json_response(conn, 422) == ControllersHelper.missing_field_response("service")
```

Comment:
Those tests look very generic. Maybe there is a way of generating them.
```elixir
@test_cases [
    {["service"], "service"},
    {["alert"], "alert"},
...
]
for {dropped, missing} <- @test_cases do
    test "Request.SendNotification.Deep.AlertNotification schema without required #{missing} field", %{ conn: conn } do
        conn =post(conn, "/v3/notification/123456", Jason.encode!
        (Map.drop(ControllersHelper.alert_request(), ["alert"])))

        assert json_response(conn, 422) == ControllersHelper.missing_field_response("alert")
    end
end
```
formatting might be fixed

Reply:
It is indeed a good idea, and since the property-based testing story is the next one in my backlog, I didn't spend so much time on it since it will be rewritten anyway.

Reply:
It is indeed a good idea, and since the property-based testing story is the next one in my backlog, I didn't spend so much time on it cause it will be rewritten anyway.

Reply:
This change should be independent of property-based testing.